README.md

```diff
@@ -252,6 +254,7 @@ Asynchronous concurrent distributed networking framework for PHP
 
 ### Promises
 
+* [amphp/amp](https://github.com/amphp/amp) - A non-blocking concurrency framework for PHP applications
```

Comment:
This is [already listed under Multitasking](https://github.com/elazar/asynchronous-php/blame/ca95cc5d900f848f55a8596c310977b1d4a41d61/README.md#L228). I understand that the framework has a promises component, but that component isn't independent of the rest of the framework, which is why I didn't also list it here.